_security/configuration/best-practices.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: default
+title: Best practices
+parent: Configuration
+nav_order: 11
+---
+
+# Best practices for OpenSearch security
+
+Setting up security in OpenSearch is crucial for protecting your data. Here are 10 best practices that offer clear steps for keeping your system safe.
+
+## 1. Use your own PKI to set up SSL/TLS
+
+Although using your own public key infrastructure (PKI), such as [AWS Certificate Manager](https://docs.aws.amazon.com/crypto/latest/userguide/awspki-service-acm.html), requires more initial effort, a custom PKI provides you with the flexibility needed to set up SSL/TLS in the most secure and performant way.
+
+### Enable SSL/TLS for node- and REST-layer traffic
+
+SSL/TLS is enabled by default on the transport layer, which is used for node-to-node communication. SSL/TLS is disabled by default on the REST layer.
+
+The following setting is required in order to enable encryption on the REST layer: 
+
+```
+plugins.security.ssl.http.enabled: true
+```
+{% include copy.html %}
+
+
+For additional configuration options, such as specifying certificate paths, keys, and certificate authority files, refer to [Configuring TLS certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls/).
+
+### Replace all demo certificates with your own PKI
+
+The certificates generated when initializing an OpenSearch cluster with `install_demo_configuration.sh` are not suitable for production. These should be replaced with your own certificates.
+
+You can generate custom certificates in a few different ways. One approach is to use OpenSSL, described in detail at [Generating self-signed certificates]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/). Alternatively, there are online tools available that can simplify the certificate creation process, such as the following:
+
+- [SearchGuard TLS Tool](https://docs.search-guard.com/latest/offline-tls-tool)
+- [TLSTool by dylandreimerink](https://github.com/dylandreimerink/tlstool)
+
+## 2. Prefer client certificate authentication for API authentication
+
+Client certificate authentication offers a secure alternative to password authentication and is more suitable for machine-to-machine interactions. It also ensures low performance overhead because the authentication occurs at the TLS-level. Nearly all client software, such as curl and client libraries, support this authentication method.
```

Comment:
```suggestion
Client certificate authentication offers a secure alternative to password authentication and is more suitable for machine-to-machine interactions. It also ensures low performance overhead because the authentication occurs on the TLS level. Nearly all client software, such as curl and client libraries, support this authentication method.
```